src/components/icons/sfrxETH.tsx

```diff
@@ -4,67 +4,28 @@ import React from 'react'
jU9uZCAlFK9k9INQDoXps+USYAESMCJBM4C6CCE6OvE4BkzCTiBAAWIE6rEGB1NQEqp/p59BeBHRyfC4EmABEhAfwKtAfQWQkj9U2WGJBA6AhQgoWNPzy4jIKVMCaCV2sTostSZLgmQAAnYnYBqItJHCHHV7oEyPhLQgQAFiA5VZA6OIiClTAWgpffK4qjgGSwJkAAJ6EPgKIBwCg99CspMnEOAAsQ5tWKkmhGQUqYA0MJ7cbO6ZvVlOiRAArYlcBDAz+oSQly3bZQMjAQ0JkABonFxmZpzCEgpGwNQG9ZfdE7UjJQESIAEHEVgB4B+QogBjoqawZKAhgQoQDQsKlNyLgEpZV0ATQHkc24WjJwESIAEbEVgHYD+QohhtoqKwZCAiwlQgLi4+EzdvgSklP8B0ARAcftGychIgARIwNYEFgAYIIT4w9ZRMjgScCEBChAXFp0pO4eAlPJtAOr1rPedEzUjJQESIIGQEpgC4BchxJyQRkHnJEAC9yVAAcKbgwQcQEBK+RqAhgDUCescJEACJEACdxNQJ5b/KoRQr1xxkAAJ2JgABYiNi8PQSOBOAlLKJwF8AaAj6ZAACZAACXgIdAUwSAhxmDxIgAScQYACxBl1YpQkcBcBKWV9AOrU3heIhwRIgARcRmAngB+FEENcljfTJQEtCFCAaFFGJuFmAlLKsgCUGFEb1zlIgARIQGcCakP570KImTonydxIQHcCFCC6V5j5uYaAlFKdIfIZgHoAHnNN4kyUBEhAdwInlOgAMFQIoc7y4CABEnA4AQoQhxeQ4ZPAvQhIKT8FoM4UKUJCJEACJOBQAosBDOf5HQ6tHsMmgSQIUIDw9iABjQlIKQsBqON9KpJC41SZGgmQgB4EbnifdijhsUyPlJgFCZDAnQQoQHhPkIALCEgp0wKo7W3l+7ILUmaKJEACziKwSbXQBTBSCHHOWaEzWhIgAX8JUID4S4zzScDhBKSUbwGo6W3n6/BsGD4JkIDDCQwCMFoIscTheTB8EiABPwhQgPgBi1NJQCcCUsoHAdQAUB1AKZ1yYy4kQAK2JhALIALAGCHEFVtHyuBIgARMIUABYgpWGiUBZxGQUuYC8AmAjwE876zoGS0JkIADCOwCMA7AWCHEFgfEyxBJgARMJEABYiJcmiYBJxLwnitSDcBHAB52Yg6MmQRIwBYELgAYry4hxGxbRMQgSIAEbEGAAsQWZWAQJGA/AlLKVF4R8iGASvaLkBGRAAnYlMBUABMARAohrto0RoZFAiQQQgIUICGET9ck4BQCUsonAFT1nrZe3ClxM04SIAHLCCwAoE4pnyiEOGqZVzoiARJwJAEKEEeWjUGTQOgISCnVHpEPAFQB8GboIqFnEiCBEBNYDmCSuoQQu0McC92TAAk4iAAFiIOKxVBJwG4EpJQ5AbwP4N8ACtgtPsZDAiRgOIEVAKaoSwixzXDrNEgCJOAKAhQgrigzkyQB8wlIKXMAeM97FTbfIz2QAAlYRCAOwJ/qEkJst8gn3ZAACWhMgAJE4+IyNRIIFQEpZXbvxnX1ZOTtUMVBvyRAAgETmAvgvwCmCiH2BWyFC0mABEjgHgQoQHhbkAAJmEpASpkRQEXv9Q6A9KY6pHESIIFACJwBMB1AtLqEEKcCMcI1JEACJOALAQoQXyhxDgmQgGEEvOeMlPc+IXnBMMM0RAIk4C+BneoJB4AYIcQsfxdzPgmQAAkESoACJFByXEcCJBA0Ae8J7OUAlAVQIWiDNEACJJAcgRkAlNiYyRPJk0PFPycBEjCLAAWIWWRplwRIwC8CUsoHAZQBUNq7b+QVvwxwMgmQwL0IbACg9nPMATBbCHGFmEiABEgg1AQoQEJdAfonARK4JwHvRvZSANRVEkBWoiIBEkiWQAKAeQBi1cUN5Mny4gQSIIEQEKAACQF0uiQBEvCfgJTyVQDqFPYSAIoBeNx/K1xBAtoR+AvAQgDzASwQQsRrlyETIgES0I4ABYh2JWVCJOAOAlLK1wEUBVDEe2VxR+bM0uUEjgBY7L0WCSHWuJwH0ycBEnAgAQoQBxaNIZMACdxNQEqZG4A6APEtAIUAvEROJKABAXXa+FIASwDECSE2a5ATUyABEnA5AQoQl98ATJ8EdCUgpXwSQEEAb3ov9eoWBwnYnYB6lWq591omhDhs94AZHwmQAAn4S4ACxF9inE8CJOBYAlLKfADyAwgDUABAXscmw8B1ILAewAoAqwCsFJu+/s8AAAIiSURBVEKs1SEp5kACJEACyRGgAEmOEP+cBEhAWwJSytQA1F6S26/XtE2YiYWSwDoAar/GrUsIcTmUAdE3CZAACYSKAAVIqMjTLwmQgC0JSClTeJ+MqKcjqvOWOo9EnU3CQQK+ElBnbqjzN1RHKvWUY70Q4oavizmPBEiABHQnQAGie4WZHwmQgCEEpJSqy1YeAC8DUBvec3kvtgM2hLDjjKj2t1u8l9oYvkld3LPhuDoyYBIggRAQoAAJAXS6JAES0IeAlPIJADm9XbdyAFDXi97f0ydR92ayFcAOANu9l+pKtVUIcdS9SJg5CZAACQRHgAIkOH5cTQIkQAL3JSClfA7A897rWQDq1894N8GTnH0IqE3gewHsAbAbwC51CSHUzzlIgARIgAQMJkABYjBQmiMBEiABXwhIKdMByA7g/wBk815Pe39PtRBWV3pfbHHOfQmcAaDa2KprH4CDAA54r/3q94QQZ8mPBEiABEjAWgIUINbypjcSIAES8JmAlDKtV4io/SfqUq97ZfZemQCo/SdqX8qjPhvVY+JJABsBqH0YxwEc817qtSh1Uri6DgshzumRLrMgARIgAb0IUIDoVU9mQwIk4GICUkr1xCSj98oA4F/eS/2+euKiLiVq1PUwgEcApPFeDwFQl2pNrF4bM3KoV5pUy9lL3usigAve6zwAdSmxoJ5GqEs9uTjtvRIBnFKXEEL9PgcJkAAJkIDDCfw/KH9XGLeVPcAAAAAASUVORK5CYII="
```

Comment:
please lets not cut corners.

you are setting a 87kb image of 800x800 inside of an svg inside of a tsx file for a 24x24 icon in the UI. This is very inefficient. As a frontend eng, you should care about file size and performance, if you add 10 icons of 87KB file will be almost 1MB and load very slow on mobile, it will also reduce SEO score.

NextJS has Image tag to optimize and put a CDN w resizing on top of images which would be more appropiated, also you could reduze and optimize the png from 87KB to probably 6-8KB.